README.md

```diff
@@ -6,7 +6,7 @@ Havala Pye, pye.havala@epa.gov
 
 ### Description
 
-The Community Regional Atmospheric Chemistry Multiphase Mechanism (CRACMM) is a description of atmospheric NO<sub>X</sub>-VOC-O<sub>3</sub>-aerosol chemistry and associated metadata (Pye et al., 2023; Place et al., 2023). CRACMM is available in the EPA [Community Multiscale Air Quality (CMAQ) modeling system](https://github.com/USEPA/CMAQ) starting with version 5.4 (2022 release). In CMAQ, CRACMM was initially released as CRACMM1 and CRACMM1AMORE which only differ in the representation of isoprene chemistry (see Wiser et al., 2023). For more information about the plans for CRACMM, see the [EPA fact sheet on the CMAQ website](https://www.epa.gov/cmaq/cmaq-fact-sheets). The main repository branch on github will continually evolve and periodically tagged to reflect official release versions. To obtain frozen versions of CRACMM, select the appropriate branch on github and/or tag. 
+The Community Regional Atmospheric Chemistry Multiphase Mechanism (CRACMM) is a description of atmospheric NO<sub>X</sub>-VOC-O<sub>3</sub>-aerosol chemistry and associated metadata (Pye et al., 2023; Place et al., 2023). CRACMM has been available in the EPA [Community Multiscale Air Quality (CMAQ) modeling system](https://github.com/USEPA/CMAQ) since version 5.4 (2022 release) and will continue to be updated (see the [EPA fact sheet on the CMAQ website](https://www.epa.gov/cmaq/cracmm)). In CMAQ, CRACMM was initially released with two representations of isoprene chemistry (see Wiser et al., 2023) that were merged together in CRACMM2 (Skipper et al., 2024). The main repository branch on github will evolve (see [Version History](#version-history)) and be periodically tagged to reflect official release versions. To obtain frozen versions of CRACMM, select the appropriate branch or tag on github. 
 
 ### Purpose and Organization of the CRACMM Github Repository
 
```

Comment:
Where it says "Chemistry: the mechanism in CMAQ-ready and markdown formats", we could add that F0AM chemistry files are available here as well.

Reply:
added F0AM mention